controller.go

```diff
@@ -841,6 +845,7 @@ addToStore:
 	if !c.isDistributedControl() {
 		c.Lock()
 		arrangeIngressFilterRule()
+		arrangeUserFilterRule()
 		c.Unlock()
 	}
 
```

Comment:
Instead of calling `arrangeUserFilterRule` two times can you move it after the `if !c.isDistributedControl() {` block ?